.devcontainer/devcontainer.json

```diff
@@ -1,79 +1,39 @@
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
 {
-  "name": "Jekyll (OpenINF Community)",
-  "build": {
-    "dockerfile": "Dockerfile"
-  },
-  "runArgs": ["--cap-add=SYS_ADMIN", "--security-opt", "seccomp=unconfined"],
-  // Configure tool-specific properties.
-  "customizations": {
-    // Configure properties specific to VS Code.
-    "vscode": {
-      // Set *default* container-specific settings.json values on container
-      // create.
-      "settings": {
-        "terminal.integrated.profiles.linux": {
-          "fish": {
-            "path": "fish"
-          }
-        },
-        "terminal.integrated.defaultProfile.linux": "fish"
-      },
+	"name": "Docker outside of Docker",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "openinf/fisher-ubuntu:latest",
 
-      // Add the IDs of extensions you want installed when the container is created.
-      "extensions": [
-        // https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments
-        "aaron-bond.better-comments",
-        // https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
-        "davidanson.vscode-markdownlint",
-        // https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
-        "dbaeumer.vscode-eslint",
-        // https://marketplace.visualstudio.com/items?itemName=dprint.dprint
-        "dprint.dprint",
-        // https://marketplace.visualstudio.com/items?itemName=editorconfig.editorconfig
-        "editorconfig.editorconfig",
-        // https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
-        "esbenp.prettier-vscode",
-        // https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree
-        "Gruntfuggly.todo-tree",
-        // https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker
-        "ms-azuretools.vscode-docker",
-        // https://marketplace.visualstudio.com/items?itemName=KatjanaKosic.vscode-json5
-        "KatjanaKosic.vscode-json5",
-        // https://marketplace.visualstudio.com/items?itemName=Shopify.theme-check-vscode
-        "Shopify.theme-check-vscode",
-        // https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
-        "streetsidesoftware.code-spell-checker",
-        // https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml
-        "tamasfe.even-better-toml",
-        // https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-remark
-        "unifiedjs.vscode-remark"
-      ]
-    }
-  },
-  // Use 'forwardPorts' to make a list of ports inside the container available
-  // locally.
-  "forwardPorts": [
-    // Jekyll server
-    4000,
-    // Live reload server
-    35729,
-    // OpenSSH server
-    2222
-  ],
+	"features": {
```

Comment:
replacing features and including them in single slim container image is preferable too

```suggestion
	"features": {
```

Reply:
![image](https://github.com/OpenINF/.github/assets/17770407/1610204f-94e4-4d7f-8da2-0be990c230c6)

&hellip;and doubled again (!)

Reply:
even doubling that was not enough:

![image](https://github.com/OpenINF/.github/assets/17770407/0da287c7-72ec-402d-86a5-86775623f91c)

course textbook min reqs are not even close either, which is odd.

i would say 256gb then!

![image](https://github.com/OpenINF/.github/assets/17770407/f280473d-8a0d-43fb-9d64-e70651ce7127)

Reply:
space has now been running out _within_ the container (not the same thing as last time)

- [Docker Engine Storage](https://docs.docker.com/storage/)

Reply:
> If you manually install Docker Engine alongside Docker Desktop, the manually installed instance uses a `daemon.json` configuration file in a different location.
>
> &mdash; https://docs.docker.com/desktop/settings/linux/#docker-engine